File: features/step_definitions/steps.js
var window_handler = require('../utils/window_handler');


module.exports = function() {
    this.Given(/^I visit "([^"]*)"$/, function (uri) {
        browser.url(uri);
        var window = new window_handler();
        window.windowResize("mobile portrait");
    });

    this.When(/^I search for "([^"]*)"$/, function (arg1) {
        browser.addValue("input[name=q]", arg1);
    });

    this.Then(/^I can see my results for "([^"]*)"$/, function (arg1) {
        browser.waitForVisible("a*=" + arg1, 5000);
    });

};

File: features/step_definitions/steps.rb
Given(/^I visit "([^"]*)"$/) do |uri|
  @browser.goto(uri)
end

When(/^I search for "([^"]*)"$/) do |arg1|
  @browser.text_field(:name => 'q').set arg1
end

Then(/^I can see my results for "([^"]*)"$/) do |arg1|
  @browser.link(:text => /#{arg1}/).wait_until_present
end
